### ISSUE TITLE
train/validation losses

### ISSUE THREAD

**Issue by tomasvanoyen** [jmather625/climatehack#5]
Hi @jmather625 , 

thanks for this great work!

I am able to run train_unet.py in your best-submission folder (on a considerable smaller gpu machine that you had .. :-) ), leading to training losses of about 2.3 (after 3 epochs). I see that this is comparable to the training loss you had in [train_unit.ipynb](https://github.com/jmather625/climatehack/blob/main/best-submission/train_unet.ipynb). My question is how does this compare to the leaderboard score of 0.879 ? Does this mean the model performance was considerably better on the leaderboard data set versus your train/test split? Or am I missing something? 

Thanks in any case. 

Regards, 

Tomas

**Comment by jmathur25** [comment]
Hi Tomas,

Referring to the table printed in `train_unet.ipynb`, I see epoch 3 had a loss of 0.2489. You reported 2.3, so it seems something is not right with your training. Can you confirm you are getting this value?

And we ultimately trained for a lot more epochs (I'm forgetting the exact number but I can dig it up if needed). The leaderboard metric was MS-SSIM, so you can roughly approximate the leaderboard score as 1 - MS_SSIM_loss. If I recall correctly, I don't think there was too much distribution shift. Leaderboard score was similar to what we got for the validation set.

**Comment by tomasvanoyen** [comment]
Hi @jmather625 , 

apologies for my typo ; indeed we have a loss of 0.23 at epoch 3, so I guess that corresponds well. 

So, I should just leave it to crunch more numbers ... 

Thanks! 

Tomas

**Comment by jmathur25** [comment]
No problem, and good luck! I'll close this for now, but feel free to reopen this or make a new issue if needed.
